clusters/build-clusters/01_cluster/openshift-image-registry/README.md

```diff
@@ -1,19 +1,37 @@
 # openshift-image-registry
 
-## Configuration
+## Enable route
 
-Doc: [expose the registry using DefaultRoute](https://docs.openshift.com/container-platform/4.2/registry/securing-exposing-registry.html#registry-exposing-secure-registry-manually_securing-exposing-registry)
+[Expose the registry using the default-route](https://docs.openshift.com/container-platform/4.2/registry/securing-exposing-registry.html#registry-exposing-secure-registry-manually_securing-exposing-registry):
 
 ```bash
-$ oc patch configs.imageregistry.operator.openshift.io/cluster --patch '{"spec":{"defaultRoute":true}}' --type=merge
+$ oc --as system:admin --context build01 patch configs.imageregistry.operator.openshift.io/cluster --patch '{"spec":{"defaultRoute":true}}' --type=merge
 
 ```
 
 TODO: add the configmap and the RBAC for `config-updater`.
 
 Pull image:
 
+```bash
+$ podman pull default-route-openshift-image-registry.apps.build01.ci.devcluster.openshift.com/ci/ci-operator:latest --tls-verify=false
+
 ```
-# podman pull default-route-openshift-image-registry.apps.build01.ci.devcluster.openshift.com/ci/ci-operator:latest --tls-verify=false
 
+## Increase route timeout
+
+[Add timeout to the route](https://docs.openshift.com/container-platform/4.3/networking/routes/route-configuration.html#nw-configuring-route-timeouts_route-configuration): Although `30s` is the default value, we still want it here to clarify this is intended
+and the value should not be over `30s`.
+
+```
+$ oc --as system:admin --context build01 annotate route default-route -n openshift-image-registry haproxy.router.openshift.io/timeout=30s
```

Comment:
We decrease the value from 1000 to 30 now.
https://coreos.slack.com/archives/C011JMD9UB1/p1587385724153500?thread_ts=1587381455.149500&cid=C011JMD9UB1

Reply:
That is the default, we do not need this step.

Reply:
removed